performance.lic

```diff
@@ -2,43 +2,54 @@
   Documentation: https://elanthipedia.play.net/Lich_script_repository#performance
 =end
 
-custom_require.call(%w[common drinfomon spellmonitor events])
+custom_require.call(%w[common common-items drinfomon spellmonitor events])
 
 class Performance
-  include DRC
 
   def initialize
     no_pause_all
     start_time = Time.new.to_i
     arg_definitions = [
       [
         { name: 'noclean', regex: /noclean/i, optional: true, description: 'Skip cleaning your instrument' },
-        { name: 'checksong', regex: /checksong/i, optional: true, description: 'Only check what song to play for best learning then exit.' }
+        { name: 'checksong', regex: /checksong/i, optional: true, description: 'Only check what song to play for best learning then exit.' },
+	{ name: 'mindstate_goal', regex: /(\d+)/, optional: true, description: 'Train until mindstate >= mindstate_goal' }
       ]
     ]
 
     args = parse_args(arg_definitions)
 
     @settings = get_settings
-    @song_list = get_data('perform').perform_options
+    song_list = get_data('perform').perform_options
+    @mindstate_goal = args.mindstate_goal.to_i
```

Comment:
```suggestion
    @mindstate_goal = args.mindstate_goal.to_i || @settings.performance_mindstate_goal
```

Reply:
And then line 107 can be more or less unchanged

Reply:
Would the command line arg take precedence over the yaml setting in that OR? My understanding is that it wouldn't then be clear which one we're using.

Reply:
Nevermind, @asechrest explained this to me.